functions/Install-DbaInstance.ps1

```diff
@@ -579,6 +579,15 @@ function Install-DbaInstance {
                 } catch {
                     Stop-Function -Message "Failed to read config file $ConfigurationFile" -ErrorRecord $_
                 }
+            } elseif ($Configuration.ACTION) {
```

Comment:
Hmm.. So if someone specifies ACTION=Install, this would not provide a full configuration, but only a few items. What if this one checked also whether provided action is NOT install?
```suggestion
            } elseif ($Configuration.ACTION -and $Configuration.ACTION -ne "Install") {
```

Reply:
Even though this change would have no impact on my use cases: Why would anyone specify a Configuration with ACTION = "Install" unless they had a very specific configuration?
Maybe someone wants to test an installation with only the bare minimum as parameters to show what the setup.exe uses as defaults. They can still fall back to use ConfigurationFile like I do now, but the command would be more flexible if they key ACTION inside of Configuration would be the switch to use a totally custom configuration file, just without the need to use a file. 

If you still like to implement you suggested change, then I will change that. No problem.

Reply:
I see where you're getting at. It would make sense to implement something like this, but the documentation of the Configuration parameter should reflect it then, as it's quite an obscure behavior.

Reply:
Good point, havn't changed documentation to just get feedback on this first. As you are generally ok with the change I will try to propose a change to the documentation during today.

Reply:
Documentation is updated. Please have a look.
If we add ` -and $Configuration.ACTION -ne "Install"` in the test I would change "As long as you don't specify the item ACTION" to "As long as you don't specify the item ACTION with a value other than 'Install'".